financial-templates-lib/logger/SpyTransport.js

```diff
@@ -9,6 +10,9 @@ class SpyTransport extends Transport {
   }
 
   async log(info, callback) {
+    // Add an `logInTest` option to help with debugging to bots in tests by printing all logs received by winston.
```

Comment:
Unrelated to main PR change. Can run tests now like:

```
truffle test ../monitors/test/index.js --network test logInTest
```
and the tests will log all winston outputs. Very useful in debugging.